assets/src/blocks/SocialMediaCards/SocialMediaCards.js

```diff
@@ -2,47 +2,48 @@ import { Component, Fragment } from '@wordpress/element';
 import { Preview } from '../../components/Preview';
 import {
   MediaUpload,
-  MediaUploadCheck
+  MediaUploadCheck,
 } from '@wordpress/editor';
 
 import {
   TextControl,
   TextareaControl,
   ServerSideRender,
   Button,
-  Tooltip
+  Tooltip,
 } from '@wordpress/components';
 
 export class SocialMediaCards extends Component {
-  constructor( props ) {
-    super( props );
-  }
-
   renderEdit() {
     const { __ } = wp.i18n;
 
     const { cards, onDeleteImage } = this.props;
 
     const getImageOrButton = ( openEvent ) => {
       if ( cards.length > 0 ) {
-
         return cards.map( ( card, index ) => (
-          <span className="img-wrap">
-                <Tooltip text={ __( 'Remove Image', 'planet4-blocks-backend' ) }>
-                  <span className="close" onClick={ ev => {
-                    onDeleteImage( card.image_id );
-                    ev.stopPropagation();
-                  } }>&times;</span>
-                </Tooltip>
-                <img
-                  src={ card.image_url }
-                  onClick={ openEvent }
-                  className="gallery__imgs"
-                  key={ index }
-                  width='150 px'
-                  style={ { padding: '10px 10px' } }
-                />
-              </span>
+          <span className="img-wrap" key={ index + 1 }>
+            <Tooltip text={ __( 'Remove Image', 'planet4-blocks-backend' ) }>
+              <span
+                className="close"
+                onClick={ ( ev ) => {
+                  onDeleteImage( card.image_id );
+                  ev.stopPropagation();
+                } }
+                role="presentation"
+              >&times;</span>
+            </Tooltip>
+            <img
+              src={ card.image_url }
+              onClick={ openEvent }
+              className="gallery__imgs"
+              key={ index }
```

Comment:
I don't think this key is still needed since the parent has one

Reply:
Yea, I added the index to the parent element, should have removed this one.